lib/ansible/modules/cloud/ovirt/ovirt_storage_domains.py

```diff
@@ -423,9 +425,10 @@ def control_state(sd_module):
 def main():
     argument_spec = ovirt_full_argument_spec(
         state=dict(
-            choices=['present', 'absent', 'maintenance', 'unattached'],
+            choices=['present', 'absent', 'maintenance', 'unattached', 'import'],
```

Comment:
We usually use past time, can you rename to 'imported'? and add a note it's supported since 2.4 in doc?